apps/gyrokinetic.c

```diff
@@ -782,7 +782,7 @@ gkyl_gyrokinetic_app_write_coll_mom(gkyl_gyrokinetic_app* app, int sidx, double
   const struct gkyl_array *fin[app->num_species];
   gk_species_lbo_moms(app, gk_s, &gk_s->lbo, gk_s->f);
   if (gk_s->lbo.num_cross_collisions)
-    gk_species_lbo_moms(app, gk_s, &gk_s->lbo, gk_s->f);
+    gk_species_lbo_cross_moms(app, gk_s, &gk_s->lbo, gk_s->f);
```

Comment:
@JonathanGorard while you are right correct this, I think we should comment this and the writing of nu_sum and nu_prim_moms out. Those quantities are only typically used for development and testing and we almost never look at them in practice, except for specific rare diagnostics. So I think we should comment out everything in `gkyl_gyrokinetic_app_write_coll_mom` having to do with nu_sum and nu_prim_moms, and we can always uncomment them when we need these detailed diagnostics.

Reply:
So, only prim_moms would be typically output?

Reply:
yeah, let's only write out prim_moms, since we actually use those to plot upar and T

Reply:
I moved things a bit so that it would be clearer what needs to be done to write out nu_sum and nu_prim_moms.